### PR TITLE
Aerostat addition and map reorganization

### PR DESCRIPTION
**Pull request by Razgriz1032**
-Adds Kara to the overmap
-Adds the aerostat to the overmap
-Created a new file structure to better organize maps for Southern Cross
-Moves fueldepot over to a new file structure

### CODE REVIEW

**Comment by Razgriz1032** [comment]
WIP because currently I am not able to get Kara or the aerostat onto the overmap. I have no idea what I am missing. I could swear that I am AT LEAST copying the fueldepot right for the aerostat.

File: maps/southern_cross/overmap/planets/kara/kara.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Overmap object for Kara, hanging in the void of space
+/obj/effect/overmap/visitable/planet/kara //Naming this /sector/kara for now, but I think it should eventually change to /planet/kara. This is just for a test right now and it should be working on live in the meantime.
+	name = "Kara"
+	desc = "The seat of NT power in the Vir system lies in the asteroid colony orbiting Kara, the NCS Northern Star. An abandoned aerostat from early Vir colonization floats on the winds of this gas giant."
+	scanner_desc = @{"[i]Stellar Body[/i]: Kara
+[i]Class[/i]: Jovian Colony
+[i]Transponder[/i]: Transmitting (NT), NanoTrasen IFF
+[i]Habitability[/i]: Low (High Temperature, Toxic Atmosphere). Habitats in orbit.
+[b]Notice[/b]: NT security is currently restricting free flights to Northern Star. Strict adherence to regular scheduling or express approval from management are the only ways in."}
+//	map_z = (Z_LEVEL_AEROSTAT)
```
Comment (Razgriz1032) [inline maps/southern_cross/overmap/planets/kara/kara.dm:10]: Commenting out temporarily. Not sure if this is required

File: maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+/turf/simulated/shuttle/wall/voidcraft/green/kara/nocol
+	color = null
+
+KARA_TURF_CREATE(/turf/unsimulated/wall/planetary)
+
+KARA_TURF_CREATE(/turf/simulated/wall)
+KARA_TURF_CREATE(/turf/simulated/floor/plating)
+KARA_TURF_CREATE(/turf/simulated/floor/bluegrid)
+KARA_TURF_CREATE(/turf/simulated/floor/tiled/techfloor)
+
```
Comment (Razgriz1032) [inline maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm:162]: Nothing below this line matters at the moment.

File: maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+	landmark_tag = "aerostat_south"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_northwest
+	name = "Aerostat - Northwest Dock"
+	landmark_tag = "aerostat_northwest"
+
+/obj/effect/shuttle_landmark/premade/aerostat/aerostat_northeast
+	name = "Aerostat - Northeast Dock"
+	landmark_tag = "aerostat_northeast"
+
```
Comment (Razgriz1032) [inline maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm:79]: Literally all of the above should be purely reflective of the fuel depot, but the aerostat is not behaving like the fuel depot:
1. It is not appearing on the overmap
2. It is not appearing on sensors for the Baby Mammoth
3. It is not appearing as an autopilot target
4. The aerostat west dock is not appearing as a docking target.

WHY

**Review by Razgriz1032 — COMMENTED** [review]
Help pls

File: maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm (hunk 3 above)
Comment (Rykka-Stormheart) [reply maps/southern_cross/overmap/planets/kara/aerostat/aerostat.dm:79]: Are the landmarks placed on the Aerostat map for each dock, and is the Visitable landmark placed on the Aerostat map?

**Pull request merged**